bebop_driver/include/bebop_driver/autogenerated/ardrone3_state_callbacks.h

```diff
@@ -870,6 +870,12 @@ class Ardrone3PilotingStateAltitudeChanged : public AbstractState
 
 };  // Ardrone3PilotingStateAltitudeChanged
 
+// Relative move ended.\n Informs about the move that the drone managed to do and why it stopped.
```

Comment:
Do you need the following class to be part of this file? Unfortunately this file is auto-generated and will be overwritten by subsequent SDK updates.